src/ex3/tasks.json

```diff
@@ -0,0 +1 @@
+[{"id":1,"content":"nadav","isCompleted":false}]
```

Comment:
List is initialized with a default nadav value for all users
![image](https://user-images.githubusercontent.com/96736630/173379809-ca1a5b3f-57cb-46b5-9ddf-6705e9d8900d.png)